grpc/cpp-cubic/CobaltGRPC.cmake

```
@@ -100,7 +104,14 @@ if(NOT COMMAND run_protoc)
 
         # Uniquify the include path list and format it properly
         list(REMOVE_DUPLICATES PROTOC_INCLUDE_LIST)
-        string(REPLACE ";" ":" PROTOC_INCLUDES "${PROTOC_INCLUDE_LIST}")
+        if(NOT WIN32)
+            string(REPLACE ";" ":" PROTOC_INCLUDES "${PROTOC_INCLUDE_LIST}")
+        else()
+            # For windows, instead of providing a single delimited list,
+            # we will just add the include flag for each path.
+            string(REPLACE ";" " -I " PROTOC_INCLUDES "${PROTOC_INCLUDE_LIST}")
+            separate_arguments(PROTOC_INCLUDES WINDOWS_COMMAND "${PROTOC_INCLUDES}")
+        endif()
```

This is a very petty preference on my part, but I think it's more readable if the `if` condition is the special case and the `else` condition is everything else:

```suggestion
        if(WIN32)
            # For windows, instead of providing a single delimited list,
            # we will just add the include flag for each path.
            string(REPLACE ";" " -I " PROTOC_INCLUDES "${PROTOC_INCLUDE_LIST}")
            separate_arguments(PROTOC_INCLUDES WINDOWS_COMMAND "${PROTOC_INCLUDES}")
        else()
            string(REPLACE ";" ":" PROTOC_INCLUDES "${PROTOC_INCLUDE_LIST}")
        endif()
```